cmd/neofs-adm/internal/modules/morph/initialize_deploy.go

```diff
@@ -206,7 +211,10 @@ func (c *initializeContext) updateContracts() error {
 	}
 
 	if err := c.sendCommitteeTx(w.Bytes(), false); err != nil {
-		return err
+		if !strings.Contains(err.Error(), common.ErrAlreadyUpdated) {
```

Comment:
It would be useful to have a function `isErrContractAlreadyUpdated(err) bool` and reuse it below.